janus/lib/client/jsx/cc-view.jsx

```diff
@@ -0,0 +1,116 @@
+import React, {useState, useEffect, useCallback, useMemo} from 'react';
+import {json_get, json_post} from "etna-js/utils/fetch";
+import {Button, Checkbox, CircularProgress, Container, FormControlLabel, Typography} from "@material-ui/core";
+import {makeStyles} from "@material-ui/core/styles";
+import DOMPurify from 'dompurify';
+import * as marked from 'marked'
+
+const useStyles = makeStyles((theme) => {
+  const tags = ["h1", "h2", "h3", "h4", "h5", "h6", "p", "b", "li", "ul"];
+  const cc = {};
+  tags.forEach((tag) => {
+    cc[`& ${tag}`] = { ...theme.typography[tag], margin: "15px 0px" };
+  });
+
+  cc["& li"]["marginLeft"] = 25;
+  cc["& ul"]["listStyle"] = "disc outside none";
+
+  return {
+    loadingRoot: {
+      minWidth: "100%",
+      minHeight: "100vh",
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "center"
+    },
+    loadingArt: {
+      display: "flex",
+      alignItems: "center"
+    },
+    cc,
+    agree: {
+      margin: "15px 25px",
+    }
+  };
+});
+
+export function CcView({project_name}) {
+  const [project, setProject] = useState(null);
+  const [agreed, setAgreed] = useState(false);
+  useEffect(() => {
+    json_get(`/api/admin/${project_name}/info`)
+      .then(
+        ({project}) => setProject(project),
+      )
+  }, []);
+
+  const classes = useStyles();
+
+  const requiresAgreement = true ? true : project ? project.requires_agreement : false;
```

Comment:
These two lines are changed once we have the janus admin ui piece in place, this just allows simpler testing with inline copy of the ipi cc.  Will remove before merging.